packages/core/src/html-entrypoint.ts

```diff
@@ -51,7 +51,7 @@ export class HTMLEntrypoint {
   }
 
   private relativeToApp(rootRelativeURL: string) {
-    return rootRelativeURL.replace(this.rootURL, '');
+    return rootRelativeURL.replace(new RegExp(`^${this.rootURL}`), './');
```

Comment:
This introduces a change, in that passing `/assets/app.js` would now return `./assets/app.js` instead of `assets/app.js`. But this was necessary to make the `resolve()` call in the other file work, otherwise `assets/app.js` would be treated like a package and not resolve correctly. 

CI seems happy, but I wonder if that could cause any unwanted side effects, that neither CI nor me understand?